### DISCUSSION TITLE
Setting DSCP at each query.

### DISCUSSION THREAD

**Discussion by yutayu** [DNSCrypt/dnscrypt-proxy#2532] [Ideas]
I want to set dscp at each query for faster response.

**Comment by jedisct1** [comment]
It's [already](https://github.com/DNSCrypt/dnscrypt-proxy/blob/0ba728b6cec395a53408250d7ab304abbe7e0d1f/dnscrypt-proxy/setsockopts_linux.go#L14) [set](https://github.com/DNSCrypt/dnscrypt-proxy/blob/0ba728b6cec395a53408250d7ab304abbe7e0d1f/dnscrypt-proxy/setsockopts_linux.go#L34).

**Comment by yutayu** [comment]
That is not Minimize-Delay .
How about Manual setting?
